pkg/logger/configure.go

```diff
@@ -63,5 +150,22 @@ func ConfigureProductionLogger(level string, syncs ...io.Writer) (Logger, error)
 		zap.Fields(zap.String("version", Version)),
 	)
 	zap.ReplaceGlobals(zapLogger)
-	return &logger{underlyingLogger: zapLogger}, nil
+
+	initialiseSentry()
+
+	traceExporter, err := newExporter(ctx)
+	if err != nil {
+		zap.L().Fatal("failed to create trace exporter", zap.Error(err))
+	}
+
+	tp := newTraceProvider(traceExporter)
+	otel.SetTracerProvider(tp)
+	tc := propagation.TraceContext{}
+	otel.SetTextMapPropagator(tc)
+
+	l := &logger{underlyingLogger: zapLogger}
+	ctx = l.InjectIntoContext(ctx)
+	ctx = tracer.NewContext(ctx, tp, "prod-logger-tracer")
```

Comment:
will this cause problems for the logger being use in our CLI tool?

Reply:
Nope - if being used in the CLI, the sentry DSN is not going to be set, meaning all of this will just work behind the scenes.